idd/Energy+.idd.in

```diff
@@ -23564,6 +23572,70 @@ AirflowNetwork:Distribution:Linkage,
       \note The zone name is where AirflowNetwork:Distribution:Component:Duct is exposed. Leave this field blank if the duct
       \note conduction loss is ignored.
 
+AirflowNetwork:OccupantVentilationControl,
+      \memo This object is used to provide advanced thermal comfort control of window opening and closing 
+      \memo for both exterior and interior windows.
+  A1, \field Name
+      \required-field
+      \type object-list
+      \object-list AirflowNetwork:OccupantVentilationControlNames
+      \note Enter the name where the advanced thermal comfort control is required.
+  N1, \field Minimum Opening Time
+      \type real
+      \units minutes
+      \minimum 0.0
+      \default 0.0
+  N2, \field Minimum Closing Time
+      \type real
+      \units minutes
+      \minimum 0.0
+      \default 0.0
+  A2, \field Thermal Comfort Low Temperature Curve Name
+      \type object-list
+      \object-list LinearCurves
+      \object-list QuadraticCurves
```

Comment:
@lgu1234 Any reason not to allow all univariatecurves? e.g.
\object-list UniVariateCurves
\object-list UniVariateTables
(and delete the linear and quadratic references)
Same for the other curve name.  If you want to restrict it to linear and quadratic, that's OK, just wanted to ask.